### PR TITLE
PMM-10333 PXC experimental dashboard

### PR DESCRIPTION
**Pull request by rasika-chivate**

### CODE REVIEW

**Review by dygksy — APPROVED** [review]

**Review by anton-bystrov — APPROVED** [review]

File: build.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # This script is used for building a themed site to preview on render.com
 # Preview URL: https://pmm-doc.onrender.com
 
-python -m pip install --upgrade pip
-pip install wheel
+# python -m pip install --upgrade pip
```
Comment (denisok) [inline build.sh:4]: @rasika-chivate is this somehow related to the change?

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+Markdown==3.3.7
```
Comment (denisok) [inline requirements.txt:1]: @rasika-chivate is this somehow related to the change?

File: build.sh (hunk 1 above)
Comment (rasika-chivate) [reply build.sh:4]: Actually not. But for successful deployment, this is required else all the builds are failing.

**Review by denisok — APPROVED** [review]
